source/ddbus/exception.d

```diff
@@ -36,7 +36,9 @@ class DBusException : Exception {
 }
 
 /++
-  Thrown when the signature of a message does not match the requested types.
+  Thrown when the signature of a message does not match the requested types or
+  when trying to get a value from a DBusAny object that does not match the type
+  of its actual value.
 +/
 class TypeMismatchException : Exception {
   package this(
```

Comment:
Maybe we should actually make these public so they can be reused in user code?

Reply:
It doesn't make much sense to make this one public, as it produces quite a specific message. I changed te other overload (the one with a `message` parameter) to be public.